README.md

```diff
@@ -1,64 +1,66 @@
-as possible. Stuff like the code that deals with the Universe and orbits will
-be clearly separated from the active game engine physics stuff. Features that
-seem important, such as rocket engines and planets, can be removed easily.
-In short, nothing will be glued together and the project is modular.
-
-Magnum isn't really much of an engine, and this project is pretty much an
-entirely new custom game engine.
-
-Build instructions and more details can be found in the
-[Wiki](https://github.com/TheOpenSpaceProgram/osp-magnum/wiki/).
-
-## Nice Engine Features
-* Uses modern **C++17**
-* Uses **entt** for an Entity Component System (this means it's fast, google it)
-* Uses **Newton Dynamics** for physics
-* Include dependencies are pretty clean (files only include what they need, and
-  there are no circular includes)
-* Compiles pretty fast
+![screenshot](screenshot0.png?raw=true "A Debug-rendered vehicle composed of parts flying over a planet.")
+
+***This project is still deep in the pre-release development phase***
+
+OpenSpaceProgram is an open source initiative with the goal of creating a space flight simulator similar to Kerbal Space Program. This project also works as a general-purpose library for space games and simulations with very large universes.
+
+Written in C++17, this project mainly features a custom game engine and a universe/orbit simulator both relying on [EnTT](https://github.com/skypjack/entt/) and [Magnum](https://github.com/mosra/magnum). The universe and game engine are synchronized for a seamless spaceflight experience from a planet's surface to deep space.
+
+By taking advantage of Entity Component System (ECS) architectures and Data-Oriented Design, this project achieves simplicity, flexibility, low coupling, and excellent performance. With these techniques in action, we can easily avoid spaghetti code and optimize for a high part count.
+
+## Features
+
+### Core
+
+* Universe
+  * Support for 'infinite' universe sizes using hierarchical Coordinate Spaces
+  * Optimal data layout to support fast N-body and patched conics simulations
+  * 64-bit integers for coordinates
+* "ActiveScene" Game Engine
+  * Scene Graph
+  * Physics interface agnostic of physics engine
+  * Configurable multipass Renderer
+  * Wiring/Connection System
+    * Virtual control systems for vehicles (routable user inputs, PID, auto-landing, ...)
+    * Resource flow
+* Asset management
+  * Uses Standard glTF as a part model format
 * Can extend bulleted lists
 
-## Things to implement (roughly in order):
-* ~~Hello World, tests, and working build system~~
-* ~~Simple console interface for streamlining~~
-* ~~Minimal Universe of Satellites~~
-* ~~Showing glTF sturdy models~~
-* ~~ActiveAreas that load nearby satellites~~
-* ~~Intergrate Newton Dynamics~~
-* ~~Controllable debug crafts that can fly around~~
-* Terrain maybe?
-* Scripting or dynamic plugin interface?
+### Extra
+
+* Newton Dynamics Physics Engine integration
```

Comment:
I'd say something like "Straight forward interface for adding any physics engine" "Newton dynamics is the currently integrated physics engine, but pull requests are welcome to integrate others"